skimage/measure/_colocalization.py

```diff
@@ -0,0 +1,295 @@
+import numpy as np
+from scipy.stats import pearsonr
+from functools import wraps
+from .._shared.utils import check_shape_equality, is_binary_ndarray
+
+__all__ = ['pearson_corr_coeff',
+           'manders_coloc_coeff',
+           'manders_overlap_coeff',
+           'intersection_coeff',
+           ]
+
+
+def pearson_corr_coeff(image0, image1, roi=None):
+    """Calculate Pearson's Correlation Coefficient between pixel intensities in
```

Comment:
```suggestion
    r"""Calculate Pearson's Correlation Coefficient between pixel intensities in
```

Currently the equations are [not rendering properly](https://3469-2014929-gh.circle-artifacts.com/0/doc/build/html/api/skimage.measure.html#pearson-corr-coeff). I think adding `r` here should fix it. Alternatively, you can omit the `r` here, but will have to use double-backslashes like `\\frac` or `\\sum` in the math code below.

Reply:
This should also resolve many of the flake8 messages like `Line 139:19: W605 invalid escape sequence '\s'` that the pep8 bot is complaining about.